magprime/templates/regextra.html

```diff
@@ -0,0 +1,29 @@
+<script type="text/javascript">
+    $(function () {
```

Comment:
The more I look at code like this, the more I wonder whether we should put some utility functions for this stuff in ``common.js``.  On the one hand, the code would be nicer and clearer to someone glancing at it if you could just say

```javascript
$('#extra_donation').insertAfterFormGroup('amount_extra');
$('#tax_exempt').appendToFormGroup('amount_extra');
```

On the other hand, what's here now is mostly just basic jQuery, so there's something to be said for that.  I dunno - not a huge deal either way.